docs/builders/clone.mdx

```diff
@@ -236,6 +236,51 @@ or responding to pattern `/dev/.+`. Example:
   - `ssd` (bool) - Drive will be presented to the guest as solid-state drive
     rather than a rotational disk.
 
+- `pci_devices` (array of objects) - Passthrough a host PCI device into the VM. For example, a graphics card or a network adapter. Devices that are mapped into a guest VM are no longer available on the host. A minimal configuration only requires either the `host` or the `mapping` key to be specifed. Note: VMs with passed-through devices cannot be migrated.
```

Comment:
Same for this docs exerpt, and the following example, they can be extracted into the common config documentation in the Go code, this way that'll be automatically compiled into a partial which you can include here.

Reply:
Done ✅